source/frontend/src/features/mapSideBar/acquisition/tabs/fileDetails/detail/models.ts

```diff
@@ -1,26 +1,38 @@
 import { ApiGen_Concepts_AcquisitionFile } from '@/models/api/generated/ApiGen_Concepts_AcquisitionFile';
 import { ApiGen_Concepts_AcquisitionFileTeam } from '@/models/api/generated/ApiGen_Concepts_AcquisitionFileTeam';
 import { formatApiPersonNames } from '@/utils/personUtils';
+import { exists, isValidId } from '@/utils/utils';
 
 export class DetailAcquisitionFile {
+  isSubFile: boolean;
   fileName?: string;
   legacyFileNumber?: string;
   assignedDate?: string;
   deliveryDate?: string;
   acquisitionPhysFileStatusTypeDescription?: string;
   acquisitionTypeDescription?: string;
+  subfileInteresttypeDescription: string | null = null;
```

Comment:
nit: consistency of capitalization between this line and the next.

Reply:
Updated